docs/components.md

```diff
@@ -55,6 +55,18 @@ Independent, small visual UI elements to use in applications.
 * [service-telemetry](https://github.com/mozilla-mobile/android-components/tree/master/components/service/telemetry) - A generic library for sending telemetry pings from Android applications to Mozilla's telemetry service.
 * [service-glean](https://github.com/mozilla-mobile/android-components/tree/master/components/service/glean) - A generic library for sending telemetry pings from Android applications to Mozilla's telemetry service (eventually replacing `service-telemetry`).
 
+## Samples
+
+* [Reference Browser (full-featured browser)](https://github.com/mozilla-mobile/reference-browser) - Advanced reference browser implementation based on the browser components.
+* [Simple Browser (simple browser)](https://github.com/mozilla-mobile/android-components/blob/master/samples/browser) - Very basic browser implementation based on the browser components.
```

Comment:
Maybe just "Simple Browser" with no parentheses is best

Reply:
Make sense. Thanks.